ddp-workspace/projects/ddp-singular/src/app/app.component.html

```diff
@@ -1,5 +1,7 @@
 <app-header></app-header>
-<router-outlet (activate)="onActivate()"></router-outlet>
+<main>
```

Comment:
Added this additional wrapper tag as to simplify layout and styles 
(avoid the common **margin-top** in children elements).